<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerView</class>
 <widget class="QWidget" name="ServerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>User:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="usersComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>To all:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="toAllTextBrowser"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Sent private:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="sentPrivateTextBrowser"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Received private:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="recivedPrivateTextBrowser"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="isBunCheckBox">
       <property name="text">
        <string>Ban user</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="deleteUserPushButton">
     <property name="text">
      <string>Delete user</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
